examples/kubernetes/ceph-osd-v1-ds.yaml

```diff
@@ -17,9 +17,12 @@ spec:
       nodeSelector:
         node-type: storage
       volumes:
-        - name: devices
-          hostPath:
-            path: /dev
+# Comment back in to use direct device access.
+# Note that this will also require enabling privileged access and
```

Comment:
The default example uses emptyDir storage. Therefore, I think the example should also run unprivileged and therefore this pull request. The comments are meant to show that if moving away from the example by using actual storage disks, the container must run privileged.

This also holds true when using a host directory for data storage. In our use case, we use all machines in a robot lab (otherwise also used by developers as desktops) to run experiments in simulation, and hence we do not have dedicated storage disks or partitions. We just want to achieve some temporary failure safety until the relevant data from the experiment has been extracted. Then we dispose of the cluster setup and can re-create it later for another experiment. In that sense we probably do not have the typical data-center storage cluster for long-running services, but use it every off and on as necessary. We use ceph mostly to get minor fail-safety (say a single machine down at a time) and to pool storage space of all nodes.